jooby/src/main/java/org/jooby/Jooby.java

```diff
@@ -3196,11 +3197,17 @@ static String logback(final Config conf) {
     if (conf.hasPath("logback.configurationFile")) {
       logback = conf.getString("logback.configurationFile");
     } else {
+      String env = conf.hasPath("application.env") ? conf.getString("application.env") : null;
+      if (env != null) {
+        URL classpathLogback = Thread.currentThread().getContextClassLoader().getResource("logback." + env + ".xml");
```

Comment:
Read the opposite, that we should avoid context class loader. Also, existing code use the `getClass().getResource` method and think we should keep using it.